tests/unit_tests/GetVariantInfoTest.php

```diff
@@ -29,103 +30,643 @@
  *
  *************/
 
+if (PHP_OS == 'WINNT') {
+    chdir('C:/Users/loesj/Documents/LUMC/git/LOVD3');
+}
+
```

Comment:
@loeswerkman Do you still need this? If so, can we solve this in some other way?